src/util/psbt/serialize.rs

```diff
@@ -370,8 +370,33 @@ fn key_source_len(key_source: &KeySource) -> usize {
 
 #[cfg(test)]
 mod tests {
+    use hashes::hex::FromHex;
     use super::*;
 
+    // Composes tree matching a given depth map, filled with dumb script leafs,
+    // each of which consists of a single push-int op code, with int value
+    // increased for each consecutive leaf.
+    pub fn compose_taproot_builder(opcode: u8, depth_map: impl IntoIterator<Item = u8>) -> TaprootBuilder {
```

Comment:
nit: Why `pub`?

Reply:
Had plans to use it in `taproot` mod to do more extensive tests